### ISSUE TITLE
Running Context Menu Actions overwrites and invalidates sf authorization

### ISSUE THREAD

**Issue by junners** [forcedotcom/salesforcedx-vscode#5049]
### Summary

Running Context Menu Actions in vscode sfdx overwrites org's authorization under ~/.sfdx/<<autheduser@authedorg>>.json

### Steps To Reproduce:

1.  Have an authorized org via sf command or use an existing one
1.  confirm that the org is connected by either `sf org list` or `cat ~/.sfdx/<<autheduser@authedorg>>.json
1.  Open Context Menu Actions by Right Clicking a file under `/classes`
1.  Deploy the specific source to org
1.  Check org statuses from `sf org list` or `cat ~/.sfdx/<<autheduser@authedorg>>.json

### Expected result

Authenticated org should stay connected

### Actual result

Authenticated org got disconnected and status says `Only absolute URLs are supported `
Under the hood, the ~/.sfdx/<<autheduser@authedorg>>.json got changed.
Context menu actions still works but you'll lose the auths and cannot use sf cli commands

### Additional information
![Prereq Screenshot](https://github.com/forcedotcom/salesforcedx-vscode/assets/6131635/04ab3ad7-f0d0-4315-9ae0-b8f6dea3f994)
![Context Menu](https://github.com/forcedotcom/salesforcedx-vscode/assets/6131635/394cc5de-625e-424a-95a2-e148cddb11e4)
![After Nuke](https://github.com/forcedotcom/salesforcedx-vscode/assets/6131635/fd3f9457-4e06-4003-b203-614d9331e27b)
![Org Display](https://github.com/forcedotcom/salesforcedx-vscode/assets/6131635/c0ba0040-0386-4e60-8cac-6530196d5641)

**Salesforce Extension Version in VS Code**: v58.13.1

**SFDX CLI Version**: @salesforce/cli/2.5.8 darwin-arm64 node-v18.14.2

**OS and version**: Mac M1 macOS Ventura 13.4.1

**Comment by CristiCanizales** [comment]
Hi @junners! Are you able to consistently reproduce this or was it a one time thing?

**Comment by junners** [comment]
Yeah it's consistent. I've been re-authing my orgs a couple of times now.

**Comment by junners** [comment]
Also seems to only happen when doing a deploy

**Comment by daphne-sfdc** [comment]
Can you try logging out of all authorized orgs?  Also, if you have any scratch orgs, delete all expired scratch orgs.  Then try redoing the authorization process.  Let me know if that works!

**Comment by CristiCanizales** [comment]
Hi @junners, sorry for the delay. As of now this is the only issue we've seen for this, so I'll close it considering that it's resolved for you. Feel free to reopen if the problem comes back.
